python/tests/test_events.py

```diff
@@ -985,3 +985,73 @@ def test_random_event_ordering():
             [N / 3, N / 3, N / 3],
             rtol=0.25,
         )
+
+
+@skip_on_valgrind
+def test_event_uses_values_from_trigger_time():
+    """For simultaneously executed events, check that values from trigger
+    times are used to compute the state update."""
+    from amici.antimony_import import antimony2amici
+
+    model_name = "test_event_vals_trig_time"
+    with TemporaryDirectory(prefix=model_name) as outdir:
+        antimony2amici(
+            r"""
+            some_time = time
+            some_time' = 1
```

Comment:
unused?

Reply:
Used. `some_time` is a RateRule target instead of an AssignmentRule target (the line above just just sets the initial value and is not an AssignmentRule), . Either would work, but this way, `some_time` will be in `x` instead of in `w`.